task/02-numbers-tasks.js

```diff
@@ -221,6 +287,11 @@ function isPrime(n) {
  *   toNumber(new Number(42), 0) => 42
  */
 function toNumber(value, def) {
+        if (parseInt(value)){
```

Comment:
Correct indentation of conditional statement blocks must be observed.
https://javascript.info/coding-style